src/components/scenes/SendConfirmationScene.js

```diff
@@ -164,7 +164,10 @@ export class SendConfirmation extends Component<Props, State> {
   componentWillUnmount () {
     this.props.reset()
     if (this.props.guiMakeSpendInfo && this.props.guiMakeSpendInfo.onBack) {
-      this.props.guiMakeSpendInfo.onBack()
+      const cb = this.props.guiMakeSpendInfo.onBack ? this.props.guiMakeSpendInfo.onBack : ''
```

Comment:
Over-complicated code. Just do: `const cb = this.props.guiMakeSpendInfo.onBack`, and the false case will be `undefined` automatically.

Also, should we add `onBack: Function` to the `GuiMakeSpendInfo` type?

Reply:
Actually, this whole addition isn't needed - the line above already checks for this case. Also, just checked, and it looks like `onBack` is already in the types, so we are good.

Reply:
Flow did not like it because onBack is optional.